src/modules/ekf_att_pos_estimator/ekf_att_pos_estimator_main.cpp

```diff
@@ -784,9 +794,9 @@ void AttitudePositionEstimatorEKF::publishAttitude()
 	_att.pitch = euler(1);
 	_att.yaw = euler(2);
 
-	_att.rollspeed = _ekf->angRate.x - _ekf->states[10] / _ekf->dtIMUfilt;
-	_att.pitchspeed = _ekf->angRate.y - _ekf->states[11] / _ekf->dtIMUfilt;
-	_att.yawspeed = _ekf->angRate.z - _ekf->states[12] / _ekf->dtIMUfilt;
+	_att.rollspeed = _LP_att_rates.apply(_ekf->angRate.x) - _ekf->states[10] / _ekf->dtIMUfilt;
```

Comment:
You need one filter per dimension. You're mixing roll/pitch/yaw into one channel here.